tests/pages/selectSalesOfferPackage.test.tsx

```diff
@@ -40,6 +42,7 @@ describe('pages', () => {
             const tree = shallow(
                 <SelectSalesOfferPackage
                     salesOfferPackagesList={selectSalesOfferPackagePropsInfoNoError.salesOfferPackagesList}
+                    productNamesList={[]}
```

Comment:
Would it make sense to pass in some product names here? As it won't ever be empty.

Reply:
it would be empty in the case where there are no products such as single fare